app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+import os
+
+
+class InvalidCommandFormat(Exception):
+    pass
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) < 3 or parts[0] != "mv":
+        raise InvalidCommandFormat(
+            "Please provide a command like 'mv source_file destination_path'"
+        )
+
+    action = parts[0]
+    source_file = parts[1]
+    destination = parts[2]
+
+    if source_file == destination and action == "mv":
+        return
+
+    if "/" not in destination and action == "mv":
```

Comment:
Do not bind script to file systems using "/" as a directory delimiter, find a cross-system way to validate that the destination contains folders.